java/src/main/java/com/logicalclocks/hsfs/FeatureGroup.java

```diff
@@ -333,10 +356,25 @@ public void commitDeleteRecord(Dataset<Row> featureData, Map<String, String> wri
     featureGroupEngine.commitDelete(this, featureData, writeOptions);
   }
 
-  public FeatureGroupCommit[] commitDetails(Integer limit) throws IOException, FeatureStoreException {
+  /**
+   * Return commit details.
+   * @param limit number of commits to return.
+   * @throws FeatureStoreException
+   * @throws IOException
+   */
+  public Map<String, Map<String,String>>  commitDetails(Integer limit) throws IOException, FeatureStoreException {
     return featureGroupEngine.commitDetails(this, limit);
   }
 
+  /**
+   * Return commit details.
+   * @throws FeatureStoreException
+   * @throws IOException
+   */
+  public Map<String, Map<String,String>>  commitDetails() throws IOException, FeatureStoreException {
+    return featureGroupEngine.commitDetails(this, null);
+  }
```

Comment:
Invert the order of these 2 functions so that they appear in the documentation from generic to specific